File: fistApp/app/src/main/java/com/example/fistapp/MainActivity.java
package com.example.fistapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle("나의 첫번째 App");
        setContentView(R.layout.activity_main);
    }
}

File: kotlinApp/app/src/main/java/com/example/kotlinapp/MainActivity.kt
package com.example.kotlinapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    lateinit var button: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button=findViewById(R.id.button);

        button.setOnClickListener {
            Toast.makeText(applicationContext, "버튼 눌림", Toast.LENGTH_SHORT).show()
        }
    }
}